src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentFields.scala

```diff
@@ -114,16 +120,47 @@ object ConsignmentFields {
 
   implicit val AddConsignmentInputType: InputObjectType[AddConsignmentInput] = deriveInputObjectType[AddConsignmentInput]()
   implicit val UpdateExportLocationInputType: InputObjectType[UpdateExportLocationInput] = deriveInputObjectType[UpdateExportLocationInput]()
+  implicit val ConnectionDefinition(_, consignmentConnections) =
+    Connection.definition[RequestContext, Connection, Consignment](
+      name = "Consignment",
+      nodeType = ConsignmentType
+    )
 
   val ConsignmentInputArg: Argument[AddConsignmentInput] = Argument("addConsignmentInput", AddConsignmentInputType)
   val ConsignmentIdArg: Argument[UUID] = Argument("consignmentid", UuidType)
   val ExportLocationArg: Argument[UpdateExportLocationInput] = Argument("exportLocation", UpdateExportLocationInputType)
+  val LimitArg: Argument[Int] = Argument("limit", IntType)
+  val CurrentCursorArg: Argument[String] = Argument("currentCursor", StringType)
 
   val queryFields: List[Field[ConsignmentApiContext, Unit]] = fields[ConsignmentApiContext, Unit](
     Field("getConsignment", OptionType(ConsignmentType),
       arguments = ConsignmentIdArg :: Nil,
       resolve = ctx => ctx.ctx.consignmentService.getConsignment(ctx.arg(ConsignmentIdArg)),
       tags = List(ValidateUserHasAccessToConsignment(ConsignmentIdArg))
+    ),
+    Field("consignments", consignmentConnections,
+      arguments = List(LimitArg, CurrentCursorArg),
+      resolve = ctx => {
+        val limit: Int = ctx.args.arg("limit")
```

Comment:
What's the maximum limit? How do we stop the client from requesting too much data at once?

Reply:
Good point. Forgot to add a maximum